modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -57,6 +57,9 @@
 # of defaulting to pandas.
 pytestmark = pytest.mark.filterwarnings(default_to_pandas_ignore_string)
 
+# Initialize env for storage format detection in @pytest.mark.xfail
+pd.DataFrame()
```

Comment:
If it is really necessary, then it will probably need to be moved to `conftest.py` file.